schema/in_memory_entity/base.json

```diff
@@ -19,6 +19,13 @@
         },
         "systemName": {
             "type": "string"
+        },
+        "systemTags": {
```

Comment:
This should be added in the web application `entity` instead.

Reply:
Done